tests/installation/bootloader_s390.pm

```diff
@@ -312,15 +332,26 @@ sub run {
     select_console("installation", timeout => 180);
 
     # We have textmode installation via ssh and the default vnc installation so far
-    if (check_var('VIDEOMODE', 'text') || check_var('VIDEOMODE', 'ssh-x')) {
-        # If libyui REST API is used, we set it up in installation/setup_libyui
-        unless (get_var('YUI_REST_API')) {
-            # Workaround for bsc#1142040
-            # enter_cmd("yast.ssh");
-            enter_cmd("QT_XCB_GL_INTEGRATION=none yast.ssh") && record_soft_failure('bsc#1142040');
+    if (!get_var('AGAMA_TEST')) {
```

Comment:
better unless and move this line above the comment, but there are two blocks repeated, that part I don't get it. I set VIDEOMODE=text only for testing, do we really need it to boot the system? could you check it?

Reply:
I have run the test without VIDEOMODE=text then it failed https://openqa.opensuse.org/tests/4419778#step/bootloader_start/64  It seems try to connect VNC and failed.

Reply:
updated code and VRs in description.